src/lib/sd2/fswatch_connections.py

```diff
@@ -133,6 +133,14 @@ def __init__(self, args, workspaces):
                    # "--exclude='\.idea/.*'",
                    # "--exclude='\.git/.*'"
                    ]
+            linux_cmd = ['--event Created',
+                         '--event Updated',
+                         '--event Removed',
+                         '--allow-overflow'
+                        ]
+            if subprocess.check_output('uname').decode('ascii').strip() == 'Linux':
```

Comment:
Please look in gen_interfaces.py for a cleaner way to detect the platform using the "platform" module

Reply:
Sure